diff -ru gdal-2.1.3-orig/frmts/jpeg2000/jpeg2000_vsil_io.cpp gdal-2.1.3/frmts/jpeg2000/jpeg2000_vsil_io.cpp >GDAL-2.1.3_patch_jasper.patch
# Patch for the JasPer support of GDAL: jas_stream_t bufbase_ is of type jas_uchar *, 
# and not every compiler understands uchar * (though it will be equivalent to
# jas_uchar * since uchar should be equivalent to unsigned char).
# Kurt Lust, March 7, 2017
--- gdal-2.1.3-orig/frmts/jpeg2000/jpeg2000_vsil_io.cpp	2017-01-20 09:20:54.000000000 +0100
+++ gdal-2.1.3/frmts/jpeg2000/jpeg2000_vsil_io.cpp	2017-03-07 16:42:06.521640041 +0100
@@ -209,7 +209,7 @@
 			/* The buffer must be large enough to accommodate maximum
 			  putback. */
 			assert(bufsize > JAS_STREAM_MAXPUTBACK);
-			stream->bufbase_ = JAS_CAST(uchar *, buf);
+			stream->bufbase_ = JAS_CAST(jas_uchar *, buf);
 			stream->bufsize_ = bufsize - JAS_STREAM_MAXPUTBACK;
 		}
 	} else {
