--- genometools-1.5.9-orig/src/annotationsketch/custom_track_gc_content.c	2016-07-21 14:32:48.000000000 +0200
+++ genometools-1.5.9/src/annotationsketch/custom_track_gc_content.c	2017-03-27 15:21:23.493176428 +0200
@@ -125,13 +125,13 @@
                           gt_graphics_get_xmargins(graphics)+ 5,
                           start_ypos
                             + gt_graphics_get_text_height(graphics)/2 - 1,
-                          "100\%");
+                          "100%%");
     gt_graphics_draw_text(graphics,
                           gt_graphics_get_xmargins(graphics)+ 5,
                           start_ypos
                             + ctgc->height
                             + gt_graphics_get_text_height(graphics)/2 - 1,
-                          "0\%");
+                          "0%%");
   }
   gt_graphics_draw_horizontal_line(graphics,
                                    gt_graphics_get_xmargins(graphics),
